cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/KafkaSource.java

```diff
@@ -65,8 +65,8 @@
   public static final String KAFKA_ZOOKEEPER = "kafka.zookeeper";
   public static final String KAFKA_BROKERS = "kafka.brokers";
   public static final String KAFKA_DEFAULT_OFFSET = "kafka.default.offset";
-  public static final String KAFKA_SCHEMA = "kafka.schema";
-  public static final String KAFKA_FORMAT = "kafka.format";
+  public static final String KAFKA_SCHEMA = "schema";
```

Comment:
If the constant value changed, then it is preferable to also change the variable name to match it